_includes/syllabus.html

```diff
@@ -1,16 +1,21 @@
 {% comment %}
   Display syllabus in tabular form.
   Days are displayed if at least one episode has 'start = true'.
+  Days are displayed as Parts if at least one episode has 'part = true'.
 {% endcomment %}
 <div class="syllabus">
   <h2 id="schedule">Schedule</h2>
 
   {% assign lesson_number = 0 %}
   {% assign day = 0 %}
   {% assign multiday = false %}
+  {% assign partnotday = false %}
   {% for episode in site.episodes %}
     {% if episode.start %}{% assign multiday = true %}{% break %}{% endif %}
   {% endfor %}
+  {% for episode in site.episodes %}
+    {% if episode.parts %}{% assign partnotday = true %}{% break %}{% endif %}
```

Comment:
According to [line 4](https://github.com/carpentries/styles/pull/170/files/fd66850de72315244eae8adc8b368d1335a08b2a#diff-6be1ef7cb9e3d5c3bd34049c20599594R4), it should be `episode.part`